projects/plugins/protect/src/js/components/admin-page/index.jsx

```diff
@@ -12,9 +20,22 @@ import { useConnection } from '@automattic/jetpack-connection';
 import Summary from '../summary';
 import VulnerabilitiesList from '../vulnerabilities-list';
 import Interstitial from '../interstitial';
+import Footer from '../footer';
 
 const Admin = () => {
-	const { isRegistered } = useConnection( { skipUserConnection: true } );
+	const { isRegistered, isUserConnected } = useConnection( { skipUserConnection: true } );
+	const { siteSuffix, adminUrl } = jetpackProtectInitialState;
+	const securityCheckoutUrl = getProductCheckoutUrl(
+		'jetpack_security_t1_yearly',
+		siteSuffix,
+		adminUrl,
+		isUserConnected
+	);
+	const learnMoreUrl = getRedirectUrl( 'jetpack-protect-footer-learn-more' );
+
+	const handleProductButton = useCallback( () => {
+		window.location = securityCheckoutUrl;
+	}, [ securityCheckoutUrl ] );
```

Comment:
sorry for asking this but I wonder whether we can land https://github.com/Automattic/jetpack/pull/24122 first and then update this section, with something like...

```es6
const { run } = useProductCheckoutWorkflow( {
	productSlug: SECURITY_BUNDLE,
	siteSuffix,
	redirectUrl,
} );
```
It continues below...

Reply:
Sure, I was expecting something was going to change after your PRs (I sent you a ping in Asana about it ;) )